tests/applications/proof.rkt

```diff
@@ -127,8 +127,7 @@
 		 (proof-unstaged prf))))
 	'timeout))
 
-(module+ benchmark
-  (run-proofs))
+(run-proofs))
```

Comment:
I think that defeats the purpose that Raffi wanted to make that module invisible if not loaded? because it's slow

Reply:
Right, I put it all in the same submodule. Before the `(bench-run ...)` heading was in the submod, but not the actual test itself that output the time. Having one but not the other in the python script was causing an error with (justly) complaining that we'd be overwriting a time and misrecording.